src/pages/Admin.js

```diff
@@ -106,17 +108,37 @@ const Code = styled.pre`
 
 const ABSTAIN = 'Abstain';
 const NOCHANGE = 'No Change';
+const DEFAULT_START = new Date();
+const DEFAULT_END = new Date(DEFAULT_START.getTime() + 7 * 24 * 60 * 60 * 1000);
+const MIN_POLL_DURATION = 24 * 60 * 60 * 1000;
+
+const calcTimeSpan = (earlier, later) => {
+  let d = Math.abs(earlier.getTime() - later.getTime()) / 1000;
+  let r = {};
```

Comment:
could we name these variables something more descriptive? also does `const` make sense here?

Reply:
I'll change those variables. After doing a little [reading](https://www.freecodecamp.org/news/constant-confusion-why-i-still-use-javascript-function-statements-984ece0b72fd/) it does not. I always felt using const gives a more rigid immutable approach for defining functions, which has its place I guess, but in agreement with that article, a lot of why I use it is because of arrow functions being the new thing. I think I might favour using `const` when defining some functions, say a filter function which could be condensed to a one-liner and would be far more elegant.